Code/Framework/AzNetworking/Tests/DataStructures/FixedSizeBitsetViewTests.cpp

```diff
@@ -42,4 +42,27 @@ namespace UnitTest
             EXPECT_FALSE(view.GetBit(0));
         }
     }
+
+    TEST(FixedSizeBitsetView, EmptyBitset)
+    {
+        AzNetworking::FixedSizeBitset<32> bitset;
+        AzNetworking::FixedSizeBitsetView view(bitset, 10, 0);
+        EXPECT_FALSE(view.GetBit(0));
+    }
+
+    TEST(FixedSizeBitsetView, TestAnySet)
+    {
+        AzNetworking::FixedSizeBitset<9> unusedBitTest(true);
+        AzNetworking::FixedSizeBitsetView view(unusedBitTest, 0, 5);
+        for (uint32_t i = 0; i < 5; ++i)
```

Comment:
nit: use bitNum (I would avoid i, j etc as var names as we not resource constrained like C)